configs/crdt/AL.yaml

```diff
@@ -4,28 +4,22 @@ links:
 - name: primary
   url: https://alpublichealth.maps.arcgis.com/apps/opsdashboard/index.html#/6d2771faa9da4a2786a509d82c8cf0f7
   overseerScript: |
-    page.manualWait();
-    await page.waitForSelector("#ember353.ember-view");
-    page.click("#ember353.ember-view");
-    await page.waitForDelay(2000);
+    await page.waitForNavigation({
+     waitUntil:"domcontentloaded"});
+    await page.waitForSelector("span:nth-of-type(14)");
+    page.click("span:nth-of-type(14)");
+    await page.waitForDelay(1000);
     page.done();
-  renderSettings: 
-    viewport:
-      height: 1400
-      width: 3000
-  message: clicking on tab 12 for AL
+  message: clicking on tab 12 (number 14)
 
 - name: secondary
   url: https://alpublichealth.maps.arcgis.com/apps/opsdashboard/index.html#/6d2771faa9da4a2786a509d82c8cf0f7
   overseerScript: |
-    page.manualWait();
-    await page.waitForSelector("#ember357.ember-view");
-    page.click("#ember357.ember-view");
-    await page.waitForDelay(2000);
+    await page.waitForNavigation({
+     waitUntil:"domcontentloaded"});
```

Comment:
yay more stable selection criteria even though it involves spans! the world we live in